SYCL/KernelParams/union_kernel_param.cpp

```diff
@@ -12,27 +12,27 @@ union TestUnion {
 public:
   int myint;
   char mychar;
-  double mydouble;
+  float myfloat;
```

Comment:
I'm fixing this file in https://github.com/intel/llvm-test-suite/pull/1246/files#diff-9f0383b6a7accb5ff82cb27e7a3b7119a9d065378c683962c9d1ebdd5e52c94f

Note that I'm updating FP literals as well.

Reply:
If any work on this PR is planned, I'd prefer you to exclude this file from it. Otherwise, if it's ready, we can submit and I'll update literals only in my PR.

Reply:
Removed in a5e1dc556260b085785915c8ef8cc4f432252dce